src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
+  width: 200px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    color: #616070;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    color: $main-text-color;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-top: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-label {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.product-card__price-value {
+  line-height: 18px;
+  font-weight: 700;
+  color: $main-text-color;
+}
+
+.product-card__buy-btn {
+  width: 100%;
+  box-sizing: border-box;
+  height: 40px;
+  background-color: $btn-color;
+  border-radius: 5px;
+  border: 1px solid $btn-color;
+  font-size: 14px;
+  font-weight: 400;
+  color: $btn-white-color;
```

Comment:
Ensure that the variable $btn-white-color is defined in your variables.scss file. If not, this will cause an error.